.github/workflows/test_ndtensors.yml

```diff
@@ -34,7 +34,8 @@ jobs:
         shell: julia --project=monorepo {0}
         run: |
           using Pkg;
-          pkg"dev ./NDTensors"
+          Pkg.develop(path=joinpath(pwd(), "NDTensors"))
+          Pkg.develop(path=pwd())
       - name: Run the tests
         run: |
-          julia --project=monorepo --depwarn=yes -e 'using Pkg; Pkg.test("NDTensors")'
+          julia --project=monorepo --depwarn=yes -e 'using Pkg; Pkg.instantiate(); Pkg.test("NDTensors")'
```

Comment:
Did this need to change?

Reply:
So I was running into a compat issue when `NDTensors` tried to compile `ITensors` in `monorepo`. But I just ran the tests again on my machine and I don't see this problem so I will revert the change